cart-service/src/connector/actions.ts

```diff
@@ -97,27 +99,43 @@ export async function createCustomTypes(
     .types()
     .get({
       queryArgs: {
-        where: `key = "${CART_METADATA}"`,
+        where: `key = "${EXTENSION_TYPES_DATA_KEY}"`,
       },
     })
     .execute();
 
+  console.log('types', types);
```

Comment:
Could you remove the console log please?

Reply:
removed